arcgis-ios-sdk-samples/Utility network/Display content of utility network container/DisplayContentUtilityNetworkContainerViewController.swift

```diff
@@ -0,0 +1,360 @@
+                contentFeatures.forEach { content in
+                    let symbol = (content.featureTable as? AGSArcGISFeatureTable)?.layerInfo?.drawingInfo?.renderer?.symbol(for: content)
+                    self.graphicsOverlay.graphics.add(AGSGraphic(geometry: content.geometry, symbol: symbol))
+                }
+                // Set the bounding box which defines the container view may be computed using the extent of the features it contains or centered around its geometry at the container's view scale.
+                if contentFeatures.count == 1,
+                   let point = contentFeatures.first?.geometry as? AGSPoint {
+                    self.mapView.setViewpointCenter(point, scale: containerElement.assetType.containerViewScale) { _ in
+                        self.presentAlert(title: nil, message: "This feature contains no associations.")
+                        guard let boundingBox = self.mapView.currentViewpoint(with: .boundingGeometry)?.targetGeometry else { return }
+                        self.identifyAssociationsWithExtent(boundingBox: boundingBox)
+                    }
+                } else {
+                    let boundingBox = AGSGeometryEngine.bufferGeometry(self.graphicsOverlay.extent, byDistance: 0.05)
+                    self.identifyAssociationsWithExtent(boundingBox: boundingBox!)
+                }
+            } else if let error = error {
+                self.presentAlert(error: error)
+            }
+        }
+    }
+    
+    /// Get associations for the specified extent and display its associations.
+    ///
+    /// - Parameter boundingBox: The gemeotry which represents the boundaries of the extent.
+    func identifyAssociationsWithExtent(boundingBox: AGSGeometry) {
+        // Add the bounding box symbol.
+        graphicsOverlay.graphics.add(AGSGraphic(geometry: boundingBox, symbol: boundingBoxSymbol.symbol))
+        let geometry = AGSGeometryEngine.bufferGeometry(graphicsOverlay.extent, byDistance: 0.05)!
+        mapView.setViewpointGeometry(geometry) { [weak self] _ in
+            guard let self = self else { return }
+            // Get the associations for this extent to display how content features are attached or connected.
+            self.utilityNetwork?.associations(withExtent: self.graphicsOverlay.extent) { containmentAssociations, _ in
+                guard let containmentAssociations = containmentAssociations else { return }
+                containmentAssociations.forEach { association in
+                    var symbol = AGSSymbol()
+                    if association.associationType == .attachment {
+                        symbol = self.attachmentSymbol.symbol
+                    } else {
+                        symbol = self.connectivitySymbol.symbol
+                    }
+                    self.graphicsOverlay.graphics.add(AGSGraphic(geometry: association.geometry, symbol: symbol))
+                }
```

Comment:
```suggestion
                let graphics: [AGSGraphic] = containmentAssociations.map { (association) in
                    let symbol = association.associationType == .attachment
                        ? self.attachmentSymbol.symbol
                        : self.connectivitySymbol.symbol
                    return AGSGraphic(geometry: geometry, symbol: symbol)
                }
                self.graphicsOverlay.graphics.addObjects(from: graphics)
```

Reply:
I tried this but it didn't add the symbols correctly. This was the output: 
![Simulator Screen Shot - iPhone 12 Pro Max - 2021-05-03 at 12 31 13](https://user-images.githubusercontent.com/52215269/116944534-8848ac00-ac2a-11eb-8a47-579da2cf1026.png)

Reply:
That's because there was a bug in my suggestion (`geometry` instead of `association.geometry`). I've updated the suggestion.